modules/exploits/windows/fileformat/shaper_pdf_bof.rb

```diff
@@ -0,0 +1,132 @@
+##
+# This module requires Metabuffer: http://metabuffer.com/download
+# Current source: https://github.com/rapid7/metabuffer-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  #Rank definition: http://dev.metabuffer.com/redmine/projects/framework/wiki/Exploit_Ranking
+  #ManualRanking/LowRanking/AverageRanking/NormalRanking/GoodRanking/GreatRanking/ExcellentRanking
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::PDF
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'    => 'PDF Shaper Buffer Overflow',
+      'Description'  => %q{
+            PDF Shaper is prone to a security vulnerability when processing PDF files. 
+            The vulnerability appear when we use Convert PDF to Image and use a specially crafted PDF file.
+	    This module has been tested successfully on Win Xp, Win 7, Win 8, Win 10.
```

Comment:
The indentation is out. Make sure you use two space soft indentation. https://github.com/rapid7/metasploit-framework/wiki/Indentation-Standards